src/main/java/duke/exception/DukeInvalidParametersException.java

```diff
@@ -0,0 +1,23 @@
+package duke.exception;
+
+public class DukeInvalidParametersException extends DukeException {
+    /**
+     * InvalidParametersException.
+     * Occurs when invalid parameters provided given valid command.
+     *
+     * @author Yap Jing Kang
```

Comment:
I think the @author tag can be placed at the top of the class along with the Javadoc to provide more consistency in the JavaDoc format! Other than that, your codes' naming and format is really clean and I enjoyed reading it!